package/cfg/rke2-cis-1.5/master.yaml

```diff
@@ -0,0 +1,1081 @@
+      tests:
+        test_items:
+          - flag: "etcd:etcd"
+            set: true
+      remediation: |
+        On the etcd server node, get the etcd data directory, passed as an argument --data-dir,
+        from the below command:
+        ps -ef | grep etcd
+        Run the below command (based on the etcd data directory found above).
+        For example, chown etcd:etcd /var/lib/etcd
+      scored: true
+      type: skip
+
+    - id: 1.1.13
+      text: "Ensure that the admin.conf file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/admin.conf; then stat -c %a /etc/kubernetes/admin.conf; fi'"
+      tests:
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 /etc/kubernetes/admin.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.14
+      text: "Ensure that the admin.conf file ownership is set to root:root (Scored) "
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/admin.conf; then stat -c %U:%G /etc/kubernetes/admin.conf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root /etc/kubernetes/admin.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.15
+      text: "Ensure that the scheduler.conf file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/scheduler.conf; then stat -c %a /etc/kubernetes/scheduler.conf; fi'"
+      tests:
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 /etc/kubernetes/scheduler.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.16
+      text: "Ensure that the scheduler.conf file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/scheduler.conf; then stat -c %U:%G /etc/kubernetes/scheduler.conf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root /etc/kubernetes/scheduler.conf
```

Comment:
If we are skipping, we need to provide a reason in the remediation. See how its done in rke1:
/var/lib/rancher/rke2/agent/pod-manifests/kube-apiserver.yaml